Program.cs

```diff
@@ -1,11 +1,14 @@
 ﻿using System.Text.Json.Serialization;
+using rexfinder_api.Migrations;
 using WebApi.Authorization;
 using WebApi.Entities;
 using WebApi.Helpers;
 using WebApi.Services;
 
 var builder = WebApplication.CreateBuilder(args);
 
+builder.Services.AddSqlite<MyPlaceDbContext>("Data Source=rexfinder_api.db");
```

Comment:
Update to "MyPlacesDbContext". Also, I would maybe make the data source name a little more specific for this table, like "MyPlaces_RexFinderDB.db" or something similar?

Reply:
I thought the data source name is the name of the entire database, not just the specific table.  If you want, I can make the change you suggest.

Reply:
I'd not change the database name.  It'll hold myplaces and users for rexfinder so just rexfinder_api.db is fine with me (or just rexfinder since it's more than just the myplaces api)